src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -497,7 +498,7 @@ void Compiler::fgEnsureFirstBBisScratch()
 }
 
 //------------------------------------------------------------------------
-// fgFirstBBisScratch: Check if fgFirstBB is a scratch block
+// fgIsFirstBBScratch: Check if fgFirstBB is a scratch block
```

Comment:
Did you intend to rename the function as well?

Reply:
Renamed it and then decided to keep the original name. Missed undoing this one.